### PR TITLE
add more RGB to FY4A

### PR DESCRIPTION
**Pull request by jhbravo**
Similar to ABI bands, I add some extra RGB composites for AGRI, today I downloaded FY4A data and test the recipes, all works fine

 - [x] Add your name to `AUTHORS.md` if not there already

### CODE REVIEW

File: satpy/etc/composites/agri.yaml [hunk 1]
```diff
@@ -6,21 +6,192 @@ composites:
...
+    description: >
+      Similar to landsat and sentinel 2 combinations for enhance vegetation
+    compositor: !!python/name:satpy.composites.SelfSharpenedRGB
+    prerequisites:
+      - name: C03
+        modifiers: [sunz_corrected, rayleigh_corrected]
+      - name: C02
+        modifiers: [sunz_corrected, rayleigh_corrected]
+      - name: green
     standard_name: true_color
```
Comment (zxdawn) [inline satpy/etc/composites/agri.yaml:197]: You should delete the last line.

**Comment by mraspaud** [comment]
@jhbravo thanks for this new PR! Don't forget to add you name to the AUTHORS.md file!

File: satpy/etc/composites/agri.yaml (hunk 1 above)
Comment (jhbravo) [reply satpy/etc/composites/agri.yaml:197]: @zxdawn basically I use the same as the true color composite, only change the input bands

**Comment by jhbravo** [comment]
> @jhbravo thanks for this new PR! Don't forget to add you name to the AUTHORS.md file!

@mraspaud done, thanks!

**Comment by mraspaud** [comment]
@jhbravo can you put you name in alphabetical order (of last names) in the AUTHORS file? After that I will merge this.

**Pull request merged**